Makefile

```
@@ -58,27 +58,36 @@ tools: protocol
-# the CLI used by exec probes).
+# Each Dockerfile expects pre-built binaries in dist/docker/<arch>/ —
+# the targets below cross-compile natively (fast — no QEMU) for HOST_ARCH
+# before invoking `docker build`. Multi-arch matrix builds happen in the
+# release workflow via the same staging pattern.
 IMAGE_REGISTRY ?= ghcr.io/latebit-io
 TAG            ?= dev
+HOST_ARCH      ?= $(shell uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/')
 
 image: image-server image-broker image-agent
 
 image-server:
```

checkmake (0.3.2)

[warning] Target body for "image-server" exceeds allowed length of 5 lines (6).

(maxbodylength)